### DISCUSSION TITLE
Creating DIA spectral library

### DISCUSSION THREAD

**Discussion by JianAtSeer** [vdemichev/DiaNN#578] [Q&A]
Hi Vadim:
Thanks for having the feature that create spectral library from DIA data. It makes the process easy and work quite well.
I have some question, how does Diann decide which spectra to add to the library for each precursor? Does it take the best scoring one from all the files? the highest abundance one? or does it generate consensus spectra from all the ms files as in some DDA tools? Thanks

cheers

Jian

**Comment by vdemichev** [comment]
Hi Jian,

Yes, all of it is based on taking the best scoring precursor ID.

Best,
Vadim
